platform/macos/app/MapDocument.m

```diff
@@ -926,8 +914,6 @@ - (NSUInteger)indexOfStyleInToolbarItem {
         [MGLStyle darkStyleURL],
         [MGLStyle satelliteStyleURL],
         [MGLStyle satelliteStreetsStyleURL],
-        [MGLStyle trafficDayStyleURL],
-        [MGLStyle trafficNightStyleURL],
```

Comment:
For consistency with iosapp, we might as well keep the styles-as-URLs here, too.